docs/agent-api.asciidoc

```diff
@@ -844,6 +844,8 @@ The optional `options` object can be used to log additional metadata with the er
 For details see the <<metadata,metadata section>>.
 
 The optional `callback` will be called after the error has been sent to the APM Server.
+It will receive an `Error` instance if the agent failed to send the error,
+and the uuid of the captured error.
```

Comment:
Better change this to just "id" as we're going away from uuid in v2 (just so we don't forget)

```suggestion
and the id of the captured error.
```